apps/fluent-tester/src/FluentTester/TestComponents/ContextualMenu/ContextualMenuTest.tsx

```diff
@@ -224,6 +224,132 @@ const nestedContextualMenu: React.FunctionComponent<{}> = () => {
   );
 };
 
+const IconContextualMenu: React.FunctionComponent<{}> = () => {
```

Comment:
A lot of this code seems to be duplicated from `nestedContextualMenu`. Is there anyway we could reuse it?  I thought maybe you could pass in the icon props to the function component or maybe just a bool `useIconButton` but I haven't gotten it to work locally.

Reply:
Yeah good idea. I'll try it!

Reply:
I actually ended up adding a simpler example for IconButton. I had to specify additional styling props as well.